guides/dev/package_managers/README.md

```diff
@@ -28,6 +28,41 @@ It is thus recommended to use of the following strategies:
 
 Both options are explained in more detail in the [Setting CKEditor Configuration](http://docs.ckeditor.com/#!/guide/dev_configuration) article.
 
+## npm
+
+This article assumes that you have **npm** already installed and added to your `PATH`. If this is not the case, please follow the [installation guide](https://docs.npmjs.com/getting-started/installing-node).
+
+### Usage
+
+In order to fetch the most recent build execute the following command:
+
+	npm install ckeditor
+
+By default CKEditor will be placed in the `node_modules/ckeditor` directory.
+
+### Adding CKEditor as Dependency Using the `package.json` File
```

Comment:
Installing npm package is pretty straightforward and this whole instruction for npm is somehow duplicating npm docs as you mentioned (btw. similar to what we have for bower and composer).

If we don't want to duplicate npm docs we can remove the whole section, but then there won't be any place in docs mentioning installation via npm. I think it is minimal and in the same time, sufficient description of npm installation. Having it, helps users so they don't have to look in other places, even simple navigation to other site, switches context and can sometimes disturb your workflow, imho. And having section with only link to other docs is probably not the most elegant solution.